src/Audio_Registers.md

```diff
@@ -179,11 +181,11 @@ The pulse channels' period dividers are clocked at 1048576 Hz, once per four dot
 This makes their sample rate equal to <math><mfrac><mn>1048576</mn><mrow><mn>2048</mn><mo>-</mo><mi>period_value</mi></mrow></mfrac></math> Hz.
 with a resulting tone frequency equal to <math><mfrac><mn>131072</mn><mrow><mn>2048</mn><mo>-</mo><mi>period_value</mi></mrow></mfrac></math> Hz.
 
-- Period value $500 means -$300, or 1 sample per 768 input cycles  
-  or (1048576 ÷ 768) = 1365.3 Hz sample rate  
+- Period value $500 means -$300, or 1 sample per 768 input cycles
+  or (1048576 ÷ 768) = 1365.3 Hz sample rate
   or (1048576 ÷ 768 ÷ 8) = 170.67 Hz tone frequency
-- Period value $740 means -$C0, or 1 sample per 192 input cycles  
-  or (1048576 ÷ 192) = 5461.3 Hz sample rate  
+- Period value $740 means -$C0, or 1 sample per 192 input cycles
+  or (1048576 ÷ 192) = 5461.3 Hz sample rate
```

Comment:
That whitespace forces line breaks. I think they are intended as stylistic choices?

Reply:
Yes, I wasn't careful with the formatter, sorry